src/main/java/org/opentripplanner/apis/gtfs/generated/GraphQLTypes.java

```diff
@@ -1,6 +1,7 @@
 // THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
 package org.opentripplanner.apis.gtfs.generated;
 
+import java.util.HashMap;
```

Comment:
You can remove this unused import. The codegen for some reason generates extra and duplicate imports in this and the datafetchers file.